codegen/bin/coins

```diff
@@ -38,5 +38,5 @@ erbs.each do |erb|
   result = template.result(binding)
 
   path = File.join(erb['folder'], erb['file'])
-  File.write(path, result)
+  File.write(path, result)  
```

Comment:
We can add a git pre-commit or pre-push hook that automatically formats/removes whitespace using something like [CLangFormat](https://clang.llvm.org/docs/ClangFormat.html) for things like this. Not entirely necessary, but perhaps something to consider.

Reply:
Agreed. Sometimes the editor add those spaces and I only notice when I push the branch.